js/interactive-guides/modules/tabbed-editor.js

```diff
@@ -282,6 +286,24 @@ var tabbedEditor = (function() {
 
             // Fill in the editors for this Tabbed Editor in different tabs
             var editors = content.editorList || [];
+
+            // Determine the width of the tabs based on the bootstrapColSize
+            // that the tabbed editor is given.
+            var numNonActiveEditors = editors.length - 1;
+            if (numNonActiveEditors < 2) {     
+                numNonActiveEditors = 1;
+                thisTabbedEditor.activeTabSize = '100%';
```

Comment:
Shouldn't the active tab size only be 100% if there is 0 non-active tabs? How will it fit if another tab is with it?

Reply:
I agree.....we will have to test to work this out.